operators/pkg/forge/ingresses.go

```diff
@@ -131,12 +139,43 @@ func IngressInstancePath(instance *clv1alpha2.Instance) string {
 	return strings.TrimRight(fmt.Sprintf("%v/%v", IngressInstancePrefix, instance.UID), "/")
 }
 
-// IngressVNCGUIPath returns the path of the ingress targeting the environment GUI vnc.
-func IngressVNCGUIPath(instance *clv1alpha2.Instance) string {
-	return strings.TrimRight(fmt.Sprintf("%v/%v/%v", IngressInstancePrefix, instance.UID, IngressVNCGUIPathSuffix), "/")
-}
-
 // IngressMyDrivePath returns the path of the ingress targeting the environment "MyDrive".
 func IngressMyDrivePath(instance *clv1alpha2.Instance) string {
 	return strings.TrimRight(fmt.Sprintf("%v/%v/%v", IngressInstancePrefix, instance.UID, IngressMyDrivePathSuffix), "/")
 }
+
+// IngressGUIPath returns the path of the ingress targeting the environment GUI vnc or Standalone.
+func IngressGUIPath(environment *clv1alpha2.Environment, instance *clv1alpha2.Instance) string {
+	switch environment.EnvironmentType {
+	case clv1alpha2.ClassStandalone:
+		return strings.TrimRight(fmt.Sprintf("%v/%v/%v", IngressInstancePrefix, instance.UID, IngressStandaloneNameSuffix+"(/|$)(.*)"), "/")
+	case clv1alpha2.ClassContainer, clv1alpha2.ClassCloudVM, clv1alpha2.ClassVM:
+		return strings.TrimRight(fmt.Sprintf("%v/%v/%v", IngressInstancePrefix, instance.UID, IngressVNCGUIPathSuffix), "/")
+	default:
+		return ""
+	}
+}
+
+// IngressGuiStatusPath returns the path of the ingress targeting the environment.
+func IngressGuiStatusPath(host string, environment *clv1alpha2.Environment, instance *clv1alpha2.Instance) string {
```

Comment:
Same here. If the function is named path, then I would return only the path (which might be also used for setting the environment variable value.